apps/files_external/lib/Lib/Storage/AmazonS3.php

```diff
@@ -742,6 +741,19 @@ public function versioningEnabled(): bool {
 		return $this->versioningEnabled;
 	}
 
+	protected function versioningEnabledNotCached(): bool {
```

Comment:
Maybe more explicit:

```suggestion
	protected function getVersioningStatusFromBucket(): bool {
```

Reply:
Great suggestion, done.

Reply:
Not done apparently

Reply:
Sorry, now it is.